packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -353,15 +292,17 @@ export class DevServerBuilder implements Builder<DevServerBuilderOptions> {
 
     config.https = true;
     if (sslKey != null && sslCert != null) {
-      config.key = sslKey;
-      config.cert = sslCert;
+      config.https = {
```

Comment:
This seems odd.  `config.https` is being set to `true` above.  Did this work before?

Reply:
It came up when I started using the correct webpack typings. I think it worked, we have a test for it. But also found it weird.